### ISSUE TITLE
Fails open in Codespaces

### ISSUE THREAD

**Issue by KEINOS** [KEINOS/Hello-Cobra#2]
The [`.devcontainer`](https://github.com/KEINOS/Hello-Cobra/tree/d3778753a62db5ec08e3757f0788702e21f6351d/.devcontainer) works on local. (macOS 10.15.7 + VSCode 1.50.1 + Remote-Containers 0.145.1 + Docker 19.03.13).

But **fails to open on GitHub Codespaces** with the below error. 

- `Failed to connect to the remote extension host server (Error: Time limit reached)`

**Comment by KEINOS** [comment]
Nope. Still fails to open.

**Comment by KEINOS** [comment]
I tried the default `.devcontainer`(Debian base-image) and it did launch in Codespaces.

But the Debian image is twice as big as ours. Default 1.6GB (Debian), Ours 800MB.

What I noticed was that in our original `.devcontainer`(Alpine base-image) there's a `${HOME}/.vscode-server` directory inside the container. But in the default(Debian) container it's `${HOME}/.vscode-remote`.

Maybe that's why it fails to connect the `remote extension host server`.

- Note: If the container is launched on `Codespaces` then the `.codespaces` file is created at the root.
